polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -78,21 +84,39 @@ use xcm_executor::{Config, XcmExecutor};
 
 pub type SessionHandlers = ();
 
-impl_opaque_keys! {
-	pub struct SessionKeys {
-		pub aura: Aura,
+mod weights;
+
+/// FIXME: this defines some types twice, might result inconsistency. Is that not a problem?
```

Comment:
Where exactly is the redundancy?

Reply:
Example:
https://github.com/integritee-network/parachain/blob/migration-preparation/polkadot-parachains/integritee-runtime/src/lib.rs#L104

and https://github.com/integritee-network/parachain/blob/migration-preparation/polkadot-parachains/integritee-runtime/src/lib.rs#L823

Reply:
Ah ok, I see. Well, only the `Header` is redefined the same, the others are different because they contain the `OpaqueExtrinsic` as type parameter instead of the real extrinsics. We could probably skip the header

Reply:
Ah, I see. I'll leave as is for now, then.